pkg/tanka/find.go

```diff
@@ -2,25 +2,26 @@ package tanka
 
 import (
 	"io/ioutil"
+	"os"
 	"path/filepath"
 
-	"github.com/grafana/tanka/pkg/jsonnet"
 	"github.com/grafana/tanka/pkg/spec/v1alpha1"
 	"k8s.io/apimachinery/pkg/labels"
 )
 
-// ListOpts are optional arguments for ListEnvs
-type ListOpts struct {
+// FindOpts are optional arguments for FindEnvs
+type FindOpts struct {
 	Selector labels.Selector
+	Name     string
 }
 
-// ListEnvs returns metadata of all environments recursively found in 'dir'.
+// FindEnvs returns metadata of all environments recursively found in 'path'.
 // Each directory is tested and included if it is a valid environment, either
 // static or inline. If a directory is a valid environment, its subdirectories
 // are not checked.
-func ListEnvs(dir string, opts ListOpts) ([]*v1alpha1.Environment, error) {
-	// list all environments at dir
-	envs, err := list(dir)
+func FindEnvs(path string, opts FindOpts) ([]*v1alpha1.Environment, error) {
```

Comment:
From my understanding, `FindEnvs` shouldn't need to be concerned with inline environment namings – it should just return all of them